rails/features/teacher_filters_instructional_materials.feature

```diff
@@ -24,8 +22,9 @@ Feature: Teacher filters instructional materials
     When I fill in "external_activity[name]" with "My grade 5 Math Activity"
     And I check "external_activity[is_official]"
     And I select "published" from "external_activity[publication_status]"
-    And under "Grade Levels" I check "5"
-    And under "Subject Areas" I check "Math"
+    And I check "grade_levels_5"
+    And I scroll "#subject_areas_math" to the center
```

Comment:
The addition of a project in default data, added content to the external_activity edit page. 
That new content shifted the location of the subject_areas below the viewport.  Locally this didn't cause a problem but on travis the subject_areas_math element couldn't be seen so the next line was failing. 

I don't understand why Capybara didn't scroll this element into view automatically, which is what I think should happen normally. But I didn't think it was worth putting more time into this.